src/documents/to-do/DeveloperIntroductionToInfusionFramework.md

```diff
@@ -4,6 +4,595 @@ layout: default
+        },
+        displayHello: {
+            type: "fluid.helloWorld.displayHello",
+        },
+        speakHello: {
+            type: "fluid.helloWorld.speakHello",
+        }
+    }
+});
+```
+
+### Summing Up
+
+This elaborate "Hello, World!" implementation has slowly introduced some of Infusion's core features. To summarize:
+
+* We implement our programs by designing [components](#components) that work together to provide the needed functionality. Components are Javascript objects adhering to certain syntax conventions that are created using the `fluid.defaults` function. While components have defaults, any of these defaults can be overriden at the time a specific instance of the component is created.
+* Components use [invokers](#invokers) to expose functionality "publicly", provide a consistent API for collaboration with other components or use by other code, and enable polymorphic behaviour when deriving new components from existing ones.
+* All components support [events and inversion of control](#events-and-inversion-of-control); these allow our programs to be built up of loosely coupled parts, and to manage sequencing through components observing and responding to their own events or the events of other components.
+* Components that need to track mutable data, state or content should be [model components](#models-and-model-components); model components can share and synchronize their data with other model components, fire events when their models are changed, and take other actions to store and respond to state changes.
+* Components that need to interact with the web page DOM to display content or interact with users are [view components](##views-and-view-components); view components are bound to specific DOM nodes when created and support the DOM Binder convention to avoid tying an implementation too tightly to specific markup.
+* Model components can use [model listeners](#listening-to-model-changes) to respond to changes in model state.
+* Components can include [subcomponents](#subcomponents-and-model-relaying), and use [model relaying](#subcomponents-and-model-relaying) to keep state synchronized between different components in larger designs. Many kinds of model relays are possible aside from two-way synchronization.
+* As program designs evolve, Infusion's configuration-oriented components make it easier to [restructure a design](#restructuring-infusion-components) by splitting out functionality into multiple components and wiring them together through IoC references.
+* When it becomes clear two components have similar behaviour, Infusion's design helps in [refactoring to share functionality](##using-invokers-for-polymorphic-behaviour-and-refactoring-to-reuse-configuration) through the use of invokers and base grades.
+* Infusion has strong supports for [multimodal implementations](#adding-further-ways-of-saying-hello) that allow programs to adapt themselves to new contexts for input and presentation.
```

Comment:
The summary needs a bit of work.  As I've stated elsewhere, by this point you should be summarizing in much less detail, and linking back to the topic if people want to refresh their memory.

You should also really end with a "next steps" section, give them an idea of what they might do to continue learning from this point.  Even something as simple as encouraging them to join the community and linking to those docs would be a good start.

I'm also noticing that we don't really make it easy to close the loop.  We should end most tutorials with some idea of how to give feedback.  If we say it only in introductory material, people are likely to forget it if they have questions or find errors while reading the actual tutorial.